server/filters/formatTimelineEvent.ts

```diff
@@ -0,0 +1,19 @@
+export default function formatTimelineEvent(value: string): string {
+  const timelineEventValue = TimelineEventValue[value as keyof typeof TimelineEventValue]
+  return timelineEventValue
+}
+
+enum TimelineEventValue {
```

Comment:
These `ENUM` values will need updating according to the content for the Timeline but this converts ENUM values into human readable text in the meantime